audb/core/load.py

```diff
@@ -899,10 +899,28 @@ def load(
         mixdown: apply mono mix-down
         sampling_rate: sampling rate in Hz, one of
             ``8000``, ``16000``, ``22500``, ``44100``, ``48000``
-        tables: include only tables matching the regular expression or
-            provided in the list
-        media: include only media matching the regular expression or
-            provided in the list
+        attachments: load only attachment files
+            for the attachments
+            matching the regular expression
+            or provided in the list.
+            Use ``[]`` to not load any attachment files
+        tables: include only tables
+            matching the regular expression
+            or provided in the list.
+            It will also limit possible media files
+            to those listed in the selected tables.
+            If ``[]`` is provided
+            only misc tables
+            used as labels in a scheme
+            and no media files are loaded
```

Comment:
```suggestion
        tables: include only tables
            matching the regular expression
            or provided in the list.
            Media files not referenced
            in the selected are automatically
            excluded, too.
            If set to ``[]`` 
            no tables and media files are loaded
            except misc tables used in schemes
```

Reply:
The exception for he misc tables applies to all possible cases not only to the `[]` case. Should I highlight this more or is it fine just applying your suggestion?

Reply:
Perfect, thanks.